website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Firebolt configurations"
+id: "firebolt-configs"
+---
+
+## Using dbt-firebolt
+
+### `quote_columns`
+
+To prevent a warning, you should add a configuration as below to your `dbt_project.yml`. For more info, see the [relevant dbt docs page](https://docs.getdbt.com/reference/resource-configs/quote_columns).
+
+
+<File name>
+```yml
+seeds:
+  +quote_columns: false  #or `true` if you have csv column headers with spaces
+```
+</File>
+
+
+### Model Configuration for Fact Tables
+
+A dbt model can be created as a Firebolt fact table and configured using the following syntax:
+
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+    { label: 'Config block', value: 'config', },
+  ]
+}>
+
+<TabItem value="project-yaml">
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  [<resource-path>]:
+    +materialized: table
+    +table_type: fact
+    +primary_index: [ <column-name>, ... ]
+    +indexes:
+      - type: aggregating
+        key_column: [ <column-name>, ... ]
+        aggregation: [ <agg-sql>, ... ]
+      ...
+```
+
+</File>
+</TabItem>
+
+<TabItem value="property-yaml">
+<File name='models/properties.yml'>
```

Comment:
Wait, this looks odd. @swanderz Users should only be applying model configurations on the model config block and the dbt_project.yml file.

Reply:
@kevinmarr @ericf-firebolt just chatted w/ @amychen1776 on this, and the dbt perspective is that while yes, you can set model configs in property files, it really isn't recommended. Her advice is to strike the property file examples for both dimension and fact tables

Reply:
Great, let's axe 'em.